routes/common.js

```diff
@@ -31,26 +28,10 @@ function ensureAuthenticated(req, res, next) {
 
 function GetRenderConfig(req) {
   var config = {};
+  let map = {'xebra': 'XeBra', 'pancake': 'PANCAKE'}
+  config.experiment = map[experiment];
```

Comment:
Where is this used except the SVG view? There the lower-case version should be used to get the right directory name